### DISCUSSION TITLE
Can't have data from BaseOfficeLocation

### DISCUSSION THREAD

**Discussion by Bounzy** [microsoft-search/pnp-modern-search#3257] [Q&A]
Hello everyone,

I used Pnp-modern-serch to create an useful People Search. I used this tutorial : 
https://www.youtube.com/watch?v=1Ny1nfU1iy0&ab_channel=Microsoft365%26PowerPlatformCommunity
I've also watched other videos to help me.

I can do all the steps. I can have results on "Jobtitle" and "Departement", but strangely I can't get any results when I do a filter on BaseOfficeLocation.
Howerver I think properties are ok : Query - Retrieve - Safe (screen1) 

![Screen1](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/dfee0d6b-b9b9-45c1-8c05-cf8d31bdea7a)

BaseOfficeLocation has also been added in the Layout Slots and the Selected properties, but it doesn't work. And I don't understand why.

Thank your help ! If you want more information I am available.
And sorry for my english and my vocabulary, I'm far from being a developer or engineer.

**Comment by wobba** [comment]
You have to share your web part setup to help out. Have you for example added `BaseOfficeLocation` to the list of properties to be returned? And what do you mean by "filter"? Do your user profiles have a value in the SPS-Location field?

**Comment by kasperbolarsen** [comment]
Works fine here, the only hickup was that I had to type in the Selected Properties combobox as the BaseOfficeLocation wasn't in the list of available properties.

**Comment by Bounzy** [comment]
By filter I mean PnP - Search Filters.

Yes BaseOfficeLocation is in the list of properties : 
![Screen2](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/7d3cdd28-a6e3-4dfa-b566-1603b5437966)
 
But for example, Department was not in the list and it works.

Yes some users have a value in the SPS-Location field : 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/7d3c4ac5-922d-4fb3-a19b-35e8d9303817)

And, when I use "India" in the SearchBox, it works : 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/e897a8b1-9942-4613-a58d-bd537e33d08f)





`You have to share your web part setup to help out. `
It seems I can't share with someone outside of my organization. I'm trying to find a way.

**Reply by wobba** [reply]
People search do no work for guest users. So what is the issue, filtering or no results from someone outside your org or both?

**Reply by Bounzy** [reply]
At the moment, I'm the only one using the search function.

The issue is filtering. In the PnP - Search Filters I can have Department and Jobtitle, but can't have Office Location : 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/7929f844-4ee5-40d9-b007-e2f5095b4a2a)


![image](https://github.com/microsoft-search/pnp-modern-search/assets/55845637/d3712e3d-d789-43a6-8e6f-583577001774)

**Comment by kasperbolarsen** [comment]
And you have verified that the users actually do have a value in the BaseOfficeLocation property using the debug layout or SharePoint Query Tool, right?

**Reply by kasperbolarsen** [reply]
and you get a result on a query like : BaseOfficeLocation:Inde  ?
